.travis.yml

```diff
@@ -22,7 +22,7 @@ before_install:
 
 install:
   - composer update $COMPOSER_FLAGS --prefer-dist
-  - npm install google-closure-library
+  - cd Resources && npm install google-closure-library && cd ../
```

Comment:
I am not sure to understand why this is needed here.

Reply:
The changing of directories? I was trying to keep it such that all node_module dependencies were in the Resources directory rather than some at the root of the project, and others in the Resource directory.